src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -33,6 +33,10 @@ public int addDishToOrder(Object inputObj) {
         return this.orders.size();
     }
 
+    public Order getOrder(int userInputInt) {
```

Comment:
Like mentioned above gotta check if index is valid for the orders array, so I think creating a method to check if index out of bounds would be useful for all the other methods as well

Reply:
ok